src/artikler/SanityArtikkel.tsx

```diff
@@ -33,15 +35,53 @@ const SanityArtikkel = (props: {slug: string; locale: "nb" | "nn" | "en"}) => {
         return (
             <Artikkel tittel="Det har oppstått en feil">
                 <Innholdstittel>Det har oppstått en feil</Innholdstittel>
-                Du kan laste siden på nytt, eller prøve igjen senere.
+                <Normaltekst>
+                    Du kan laste siden på nytt,{" "}
+                    <Lenke href="https://www.nav.no/">gå til forsiden</Lenke>,
+                    eller prøve igjen senere .
+                </Normaltekst>
+
+                <Undertittel>In English</Undertittel>
+                <Normaltekst>
+                    An error occured. You can try to refresh the page, Go to the{" "}
+                    <Lenke href="https://www.nav.no/">front page</Lenke>, or try
+                    again later.
+                </Normaltekst>
             </Artikkel>
         );
     }
 
     if (notFound) {
         return (
-            <Artikkel tittel="Denne siden finnes ikke">
-                <Innholdstittel>Denne siden finnes ikke</Innholdstittel>
+            <Artikkel tittel="Fant ikke siden">
+                <Innholdstittel>Fant ikke siden</Innholdstittel>
+                <Normaltekst>
+                    Beklager, siden kan være slettet eller flyttet, eller det
+                    var en feil i lenken som førte deg hit.
```

Comment:
Forslag:
Beklager, siden kan være slettet, flyttet, eller det var en feil i lenken som førte deg hit.
(Pirk - må ikke gjøres - fikser i OCD med ny linje og setning med flere eller i seg)

Reply:
Hvis selve teksten blir mer lesbar av å endre den er jeg for. :) Men det kan fort bli litt uheldig hvis vi skal utforme alle tekstene våre etter hvordan de brekker på siden. Tekstene vil jo også brekke annerledes på eks mobil og nettbrett. :)

Reply:
Setninger med flere "eller" i seg kan være kronglete å lese synes jeg

Reply:
Setninger med flere "eller" i seg kan være kronglete å lese synes jeg.
(Oppramsing av feil vs. to leddsetninger med feil)
(Fortsatt pirk)